src/meshes/mesh_io.jl

```diff
@@ -220,6 +220,13 @@ function save_mesh_file(mesh::P4estMesh, output_directory, timestep, mpi_paralle
     return filename
 end
 
+# TODO: Implement this function as soon as there is support for this in `t8code`.
+function save_mesh_file(mesh::T8codeMesh, output_directory, timestep, mpi_parallel)
+    @warn "Mesh file output not supported yet for `T8codeMesh`."
+
+    return joinpath(output_directory, "dummy_mesh.h5")
+end
```

Comment:
I'm not sure whether this should be a warning or an error. What is the reason for your choice?